### PR TITLE
Added text to Start page as requested by Katherine

### PR DESCRIPTION
**Pull request by murrayholmes96**
Added text to start page
Issue #484

### CODE REVIEW

**Comment by murrayholmes96** [comment]
#484

File: src/views/start.njk [hunk 1]
```diff
@@ -4,21 +4,72 @@
...
+      }) }}
       <form method="post">
         {{ govukButton({
           text: "Start now",
           classes: "govuk-!-margin-top-2 govuk-!-margin-bottom-8 naturescot-forward-button",
           isStartButton: true
         }) }}
       </form>
+      <p class="govuk-body">
+      If you need any assistance with this registration, vontact the NatureScot Liscensing Team.
```
Comment (paulcoxsnh) [inline src/views/start.njk:52]: Should be `contact`?

**Review by paulcoxsnh — CHANGES_REQUESTED** [review]
Just one little typo, the rest all looks good :)

File: src/views/start.njk (hunk 1 above)
Comment (paulcoxsnh) [reply src/views/start.njk]: Resolved in https://github.com/Scottish-Natural-Heritage/trap-registration-return/pull/7/commits/f0334437e70fd46a2e2c5abc78b1c6645feab2b4

**Review by paulcoxsnh — APPROVED** [review]
All looks good :)

**Pull request merged**